x-pack/qa/third-party/active-directory/src/test/java/org/elasticsearch/xpack/security/authc/ldap/ActiveDirectoryGroupsResolverTests.java

```diff
@@ -53,6 +64,33 @@ public void testResolveSubTree() throws Exception {
                 containsString("Supers")));
     }
 
+    @SuppressWarnings("unchecked")
+    public void testResolveTokenGroupsSID() throws Exception {
```

Comment:
I think my original suggestion as to where we should put this test was misleading, sorry for that.  Probably `ActiveDirectorySessionFactoryTests` is a more proper place for this to live since it has to do with a specific attribute being properly added as metadata ( even though the attribute _is_ groups related )

Reply:
Moved